docs/reference/ImageDraw.rst

```diff
@@ -403,6 +403,16 @@ Methods
 
     Return the size of the given string, in pixels.
 
+    Use :py:meth:`textlength()` to measure the offset of following text with
```

Comment:
?
```suggestion
    Use :py:meth:`textlength()` to measure the offset of given text with
```

Reply:
"following" is correct, although perhaps there is a better way to word this?

This sentence is supposed to highlight the difference between `textsize` and `textlength`:
* `textsize` returns the size of the bounding box (except for the height inaccuracy for historical reasons).
   This describes the width of **this** section of text.
* `textlength` returns the distance the text cursor should move by, before rendering the next section of text.
   This describes the width of text for word-wrapping or styling purposes, and where to draw the **next** section.